webforces/server/structs.py

```diff
@@ -46,7 +46,7 @@ def fromDict(cls, dict):
 
 @dataclass
 class Algorithm:
-    alg_id: int
+    alg_id: int = -1
```

Comment:
This is done intentionally. You should provide an ID. If you don't have any just use 0.